modules/odc-using-the-developer-catalog-to-add-services-or-components.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.adoc
+
+[id="odc-using-the-developer-catalog-to-add-services-or-components_{context}"]
+= Using the Developer Catalog to add services or components to your application
+
+Developer Catalog enables you to create applications and services based on Operator backed services and templates such as Databases, Builder Images, and Helm Charts. You can, add shared applications, services, event sources, or source-to-image builders to your project using the Developer Catalog. Cluster administrators can customize the content made available in the catalog.
```

Comment:
The Developer Catalog...
Suggest: The Developer Catalog contains a collection of application components, services, event sources, or source-to-image builders that you can add to your project.

Reply:
Thanks for the suggestion. Will incorporate this.